alf/examples/muzero_atari_conf.py

```diff
@@ -310,17 +444,28 @@ def _scale_grad(scale):
         else:
             return [lambda x: torch.lerp(x.detach(), x, scale)]
 
+    rv_weight_opt_args = dict(
+        weight_decay=rv_weight_decay,
+        l2_regularization=rv_weight_l2,
+        fixed_norm=False,
+        max_norm=last_weight_max_norm)
+
+    rv_bias_opt_args = dict(
+        weight_decay=rv_bias_decay, l2_regularization=rv_bias_l2)
+
     value_net = alf.layers.Sequential(
         partial(_summarize_grad, name='value_grad'),
         *_make_trunk(),
         *_scale_grad(1 / num_quantiles if scale_grad_by_num_quantiles else 1),
-        alf.layers.FC(
-            dim,
-            num_quantiles,
-            weight_opt_args=dict(weight_decay=rv_weight_decay),
-            bias_opt_args=dict(weight_decay=rv_bias_decay),
-            bias_initializer=_get_rv_bias_initializer(),
-            kernel_initializer=torch.nn.init.zeros_),
+        alf.layers.AMPWrapper(
```

Comment:
Is this because the sensitivity to precision? Want to confirm this is independent of whether we use `NeroPlus` or not. 
Also suggesting adding comments here in case someone wonders the choice of explicitly disabling AMP when reading this.